tests/download_test.py

```diff
@@ -51,3 +64,15 @@ def test_stream(self):
                         os.path.join(path, self.test_hash_value))[0],
                     expected_return_code=0,
                 )
+
+    def test_stream_no_results(self):
+        with file_utils.temp_dir(dict(malicious=self.test_eicar)) as (path, files):
```

Comment:
What's the point of `eicar` here?

Reply:
`eicar` was being used for tests. You can simply disable the mocking and run this against `e2e` in order to get real results, which in turn I used to generate the mocks. I did not write tests for empty results though, and I think we could remove it here.